interface-elements-for-desktop/articles/rich-text-editor/footnotes-and-endnotes/change-note-appearance.md

```diff
@@ -0,0 +1,44 @@
+# Change Note Appearance
+
+Click the **Footnotes** dialog box launcher on the **References** tab or right-click a note and select **Note Options**.
+
+![invoke ribbon](../../../images/richedit-dialogs-footnote-and-endnote-invoke.png)
+
+![invoke context menu](../../../images/richedit-dialogs-footnote-and-endnote-invoke-context-menu.png)
+
+This invokes the **Footnote and Endnote dialog**. Options in this dialog allow you to specify note location, layout, and format.
+
+![dialog](../../../images/richedit-dialogs-footnote-and-endnote.png)
+
+To specify what note options to change, click the **Footnotes** or **Endnotes** radio button in the dialog's **Location** section.
+
+You can apply changes to the current section or to the whole document. Select the required options in the **Apply changes to:** drop-down list.
+
+![apply to](../../../images/rich-edit-footnotes-and-endnotes-apply-to.png)
+
+## Change Note Location
+
+Select the desired note location from **Location** the drop down menu. You can place footnotes at the bottom of the page or below text. Endnotes can be located at the end of the document or at the end of each section.
```

Comment:
What do you mean by "from **Location** the drop down menu"? (Please fix this typo.)